BaragonCore/src/main/java/com/hubspot/baragon/models/BaragonSource.java

```diff
@@ -0,0 +1,51 @@
+package com.hubspot.baragon.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Optional;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BaragonSource {
+  private final String name;
+  private final Optional<Integer> port;
+
+  @JsonCreator
```

Comment:
@tpetr I've added `fromString` method. The reason I wrote this low level migration is that if in future `BaragonSource` class changes and you want to support all migrations there is an possibility that you'd need to change existing migration. I am happy to change it to use `BaragonSource.fromString` if you feel that its right way to do it.